core/smart_contract.go

```diff
@@ -218,19 +217,18 @@ func (c *Core) FetchSmartContract(requestID string, fetchSmartContractRequest *F
 	binaryCodeFileDestPath := filepath.Join(binaryCodeFilePath, "binaryCodeFile")
 
 	// Read the content of binaryCodeFile
-	binaryCodeContent, err := ioutil.ReadAll(binaryCodeFile)
+	binaryCodeContent, err := io.ReadAll(binaryCodeFile)
 	if err != nil {
 		c.log.Error("Failed to read binary code file", "err", err)
 		return basicResponse
 	}
```

Comment:
Set `basicResponse.Message` string and replicate for other `if err` blocks that follows